lib/chat/messages/room_key_copy.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Chat.Messages.RoomKeyCopy do
+  @moduledoc "Room invite message"
+
+  @type t() :: %{room_identity: map(), timestamp: integer()}
+  defstruct room_identity: nil, timestamp: 0
+
+  def new(identity) do
```

Comment:
How this differs to  `room_invite`?

Reply:
In DB - it doesn't. But there should be difference in rendering I guess